starters/create-builder/nextjs/pages/[[...page]].tsx

```diff
@@ -63,9 +65,6 @@ export default function Page({
       <Head>
```

Comment:
I think we should keep the  `  <meta name="robots" content="noindex" />` in the server rendered string, if no page:

```

  return (
    <>
      <Head>
       { !page &&   <meta name="robots" content="noindex" />}
        <meta name="viewport" content="width=device-width, initial-scale=1" />
      </Head>
      <BuilderComponent model="page" content={page} />
    </>
  );


```